modules/templates/templates/styles/templates.module.css

```diff
@@ -82,4 +60,19 @@
 .paginationButton:disabled {
   opacity: 0.5;
   cursor: not-allowed;
+}
+
+.input {
+  width: 100%;
+  padding: 8px 12px;
+  font-size: 14px;
+  border: 1px solid #e2e8f0;
+  border-radius: 4px;
+  background-color: #f8fafc;
+}
+
+.input:focus {
+  outline: none;
+  border-color: #38b2ac;
+  box-shadow: 0 0 0 3px rgba(56, 178, 172, 0.1);
```

Comment:
Input CSS in a template page?

We shall use the input resources, not remake a new input every time
If is a special resource table.input.ts then create a new one